src/config/lang/english.go

```diff
@@ -262,6 +262,19 @@ const (
 	CmdToolsSbomShort = "Generates a Software Bill of Materials (SBOM) for the given package"
 	CmdToolsSbomErr   = "Unable to create sbom (syft) CLI"
 
+	CmdToolsWaitForShort = "Waits for a given Kubernetes resource to be ready"
```

Comment:
There are linting errors above this that would be easy to fix while you are here.

Reply:
Indeed... 😆 

![Screenshot 2023-02-21 at 11 41 52 AM](https://user-images.githubusercontent.com/882485/220419993-af46b94a-fe3e-4c18-8a3d-82a8ce84c0ee.png)